art/api/tests/tests_models.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+
+from ..models import AssetType
+
+class AssetTypeModelTest(TestCase):
+    """Tests for the Asset Types Model"""
+    def setUp(self):
+        test_asset = AssetType(type="TestAsset")
+        test_asset.save()
+
+    def test_add_new_asset_type(self):
+        """Test add new asset type in model"""
+        new_assettype = AssetType(type="Macbook")
+        new_assettype.save()
+        all_assettypes = AssetType.objects.all()
+        self.assertEqual(len(all_assettypes), 2)
```

Comment:
we should also check before and after creation to confirm an increase in asset count. (you can also do `AssetType.objects.count()` instead of fetching all then getting the `len`)

Reply:
Will do